src/test/java/core/basesyntax/service/FileWriterTest.java

```diff
@@ -8,20 +8,26 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import org.junit.Before;
 import org.junit.Test;
 
 public class FileWriterTest {
     private static final String WRITER_FILE_PATH = "src/test/resources/output.csv";
     private static final String WRONG_PATH = "src/test/resources";
-    private static final FileWriter fileWriter = new FileWriterImpl();
-    private final String report = "I'm not a Superman" + System.lineSeparator() + "...";
+    private static final String REPORT = "I'm not a Superman" + System.lineSeparator() + "...";
+    private FileWriter fileWriter;
+
+    @Before
+    public void start() {
+        fileWriter = new FileWriterImpl();
+    }
```

Comment:
should this be done before each test?

Reply:
We have two test methods, and code will be dublicate without @Before annotation.